src/main/java/org/broadinstitute/hellbender/metrics/QualityYieldMetricsArgumentCollection.java

```diff
@@ -17,6 +17,6 @@
             fullName = StandardArgumentDefinitions.USE_ORIGINAL_QUALITIES_LONG_NAME,
             doc = "If available in the OQ tag, use the original quality scores " +
                     "as inputs instead of the quality scores in the QUAL field.")
-    public boolean useOriginalQualities = false;
+    public boolean useOriginalQualities = true;
```

Comment:
Turns out that in picard this defaults to true, which is likely why @cmnbroad changed it. Setting it back to true.